docs/kyma/docs/040-cr-installation.md

```diff
@@ -26,14 +26,24 @@ metadata:
 spec:
   version: "1.0.0"
   url: "https://sample.url.com/kyma_release.tar.gz"
+  components: 
+    - name: "remote-environments"
+      namespace: "kyma-integration"
```

Comment:
i was trying to add generic, kyma-style example showing all "component features" at all and not putting whole real list of components here (cause as you pointed out it will be outdated at some point). There's note saying where full and up to date list can be found (please note that we can not use linking to our doc). For more detailed examples, use-cases, how-to's we should have separate docu explaining them in details (e.g. updating kyma-installation, adding  new component and so on)